Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -699,12 +699,11 @@ static void setViewportMinMaxDepthIntoBuffer(auto& fragmentDynamicOffsets, float
     if (!minVertexCount || !minInstanceCount || indexBufferOffsetInBytes >= indexBuffer.length || -baseVertex > static_cast<int64_t>(minVertexCount))
         return std::make_pair(IndexCall::Skip, nil);
 
-    id<MTLBuffer> indexedIndirectBuffer = apiIndexBuffer->indirectIndexedBuffer();
     auto primitiveOffset = primitiveType == MTLPrimitiveTypeLineStrip || primitiveType == MTLPrimitiveTypeTriangleStrip ? 1u : 0u;
     auto effectiveMinVertexCount = minVertexCount == RenderBundleEncoder::invalidVertexInstanceCount ? (minVertexCount - primitiveOffset) : (baseVertex + minVertexCount);
     if (apiIndexBuffer->canSkipDrawIndexedValidation(firstIndex, indexCount, effectiveMinVertexCount, indexType, firstInstance)) {
         if (apiIndexBuffer->didReadOOB())
-            return std::make_pair(IndexCall::CachedIndirectDraw, indexedIndirectBuffer);
+            return std::make_pair(IndexCall::Skip, nil);
```

Comment:
What would otherwise be OOB drawIndexed calls are allowed to be skipped completely by the specification

Reply:
Also there is no behavior change. Previously the indirect buffer would look like:
```
       {
            .indexCount = 0u,
            .instanceCount = 0u,
            .indexStart = firstIndex,
            .baseVertex = baseVertex,
            .baseInstance = firstInstance
        },
```
that is the same as skipping, effectively